src/SharpCompress/Common/ArchiveEncoding.cs

```diff
@@ -36,12 +36,10 @@ public ArchiveEncoding(Encoding def, Encoding password)
             Password = password;
         }
 
-#if !NET461
```

Comment:
@adamhathcock I removed this `ifdef`. Tests and build still worked fine.

Should I put it back or broaden it to be excluded for all `NETFRAMEWORK`?

Reply:
Ok, I'll put this back

Reply:
done